src/asciidoc/docs/guides-general/enable-file-uploads.adoc

```diff
@@ -9,19 +9,19 @@ The @withkoji/core package provides two Frontend classes for file uploads: <<cor
 
 This topic helps you choose the right controls for specific file types and scenarios.
 
-* <<_only_one_media_type,If your app accepts only one media type (audio, image, or video).>>
-** <<_only_audio_files,Only audio files.>>
-** <<_only_images,Only images.>>
-** <<_only_videos,Only videos.>>
-* <<_two_or_more_media_types,If your app accepts two or more media types (audio, image, or video).>>
-* <<_any_file_as_is,If your app accepts any file as is, typically for downloading (not rendered).>>
-* <<_generated_content,If your app generates new content (audio, image, or video) that needs to be saved.>>
+* If your upload control accepts <<_only_one_media_type,only one media type>>:
```

Comment:
Yes, much better! I would just remove the link for "only one media type", since you already link to the specific subsections of interest.

Reply:
I thought of doing that too, but I decided against it, because there's important information in the part above the subsections. Won't fix.